orc8r/cloud/go/services/tenants/servicers/storage/storage_test.go

```diff
@@ -161,3 +165,35 @@ func TestBlobstoreStore_DeleteTenant(t *testing.T) {
 	err = s.DeleteTenant(0)
 	assert.EqualError(t, err, "error")
 }
+
+// TODO(reginawang3495): Do or make ticket for making other tests in this file use less mocking
```

Comment:
xx

Reply:
This amount of testing seems sufficient 👍